visualizer/semanticsEditor.js

```diff
@@ -360,6 +366,177 @@
   }
   ohmEditor.parseTree.addListener('cmdclick:traceElement', toggleSemanticsEditor);
 
+  // Retrieve arguments from the menu entry's submenu
+  function retrieveArgumentsFromSubList(entry) {
+    var argumentWrappers = entry.querySelectorAll('ul li');
+    if (!argumentWrappers) {
+      return undefined;
+    }
+
+    var operationArguments = Object.create(null);
+    Array.prototype.forEach.call(argumentWrappers, function(argumentWrapper) {
+      var name = argumentWrapper.querySelector('.name').textContent;
+      var valueContainer = argumentWrapper.querySelector('.value');
+      var value = eval(valueContainer.value);    // eslint-disable-line no-eval
+      operationArguments[name] = value;
+    });
+    return operationArguments;
+  }
+
+  // Create a menu entry that wraps the argument name and a textarea for user to enter
+  // corresponding value.
+  function createArgumentWrapper(name) {
+    var argumentWrapper = domUtil.createElement('li');
```

Comment:
I think this code would be slightly easier to read if you use a shorter name for this variable. Since we're already in a function called `createArgumentWrapper`, you could probably just call it `wrapper`.